app/src/menu.js

```diff
@@ -1,124 +1,135 @@
-import electron from 'electron'; // eslint-disable-line import/no-extraneous-dependencies
+const electron = require('electron'); // eslint-disable-line import/no-extraneous-dependencies
 
 const { Menu } = electron;
 
-const template = [
-  {
-    label: 'Edit',
-    submenu: [
-      {
-        role: 'undo',
-      },
-      {
-        role: 'redo',
-      },
-      {
-        type: 'separator',
-      },
-      {
-        role: 'cut',
-      },
-      {
-        role: 'copy',
-      },
-      {
-        role: 'paste',
-      },
-      {
-        role: 'selectall',
-      },
-    ],
-  },
-  {
-    label: 'View',
-    submenu: [
-      {
-        role: 'reload',
-      },
-      {
-        role: 'togglefullscreen',
-      },
-    ],
-  },
-  {
-    label: 'Window',
-    submenu: [
-      {
-        role: 'minimize',
-      },
-    ],
-  },
-  {
-    label: 'Help',
-    submenu: [
-      {
-        label: 'Lisk Website',
-        click() {
-          electron.shell.openExternal('https://lisk.io');
+const buildTemplate = i18n =>
+  [
+    {
+      label: i18n.t('Edit'),
+      submenu: [
+        {
+          role: 'undo',
+          label: i18n.t('Undo'),
         },
-      },
-      {
-        label: 'Lisk Chat',
-        click() {
-          electron.shell.openExternal('https://lisk.chat');
+        {
+          role: 'redo',
+          label: i18n.t('Redo'),
         },
-      },
-      {
-        label: 'Lisk Explorer',
-        click() {
-          electron.shell.openExternal('https://explorer.lisk.io');
+        {
+          type: i18n.t('separator'),
```

Comment:
This should be just `type: 'separator',`.  'separator' is not a string in the UI.